core/src/test/kotlin/fr/sncf/osrd/utils/DummyInfra.kt

```diff
@@ -70,6 +70,20 @@ class DummyInfra : RawInfra, BlockInfra {
         return id
     }
 
+    /** Creates a route made of several blocks */
```

Comment:
```suggestion
    /** Creates a path made of several blocks */
```

`Route` would imply something different here

Reply:
done